libs/client/cornflow_client/core/instance.py

```diff
@@ -12,8 +16,25 @@ class InstanceCore(InstanceSolutionCore, ABC):
     The instance template.
     """
 
-    # TODO: make abstractmethod
-    def check(self, *args, **kwargs) -> dict:
+    def data_checks(self) -> dict:
+        """
+        Method that executes the InstanceCore.check() method and validates the result against the schema_checks
+
+        :return: The dictionary returned by the InstanceCore.check() method
+        :rtype: dict
+        :raises BadInstanceChecks: if the instance checks do not match the schema
+        :author: baobab soluciones
+        """
+        checks = self.check()
+        validator = Draft7Validator(self.schema_checks)
+        if not validator.is_valid(checks):
```

Comment:
It would be interesting to include a warning message when, despite the schema validation being correct, it is detected that the tables or columns of the json being validated are not in the schema. And print the tables or columns that are not being checked. 
This applies to all schema validations.

Reply:
Can you create an issue with this, because I do not think with the current jsonschema package taht we use is not possible but I will checkj it out, so I want to close this PR without this.